### PR TITLE
Liam_Cohen_hw1

### PR DESCRIPTION
**Pull request by liamcharlescohen**
I tried to do RMarkdown and was unsuccessful at first. I added some information about myself i.e.my degree and academic interests. 

I struggled with RMarkdown at first because I was very confused on how to make any changes. But after after watching the video (and especially the explanation of how to commit and push information), I was able to add all the required features, such as links, headers, emphasis, and a list. 

If you search things on google it is especially easy to understand formatting in R!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+## Who am I? 
+My name is Liam Cohen. I am currently a **rising third year** in the College
+majoring in *Latin American Studies and Political Science*. I highly recommend 
+that you check out the 
+[Center for Latin American Studies](https://clas.uchicago.edu/).
```
Comment (cheyennewh) [inline README.md:6]: Cool use of the link function!

**Review by cheyennewh — COMMENTED** [review]
Nice to *virtually* meet you! In this HW you used a lot of the elements that were talked about in lecture and in the HW suggestions. I like the way you formatted this file and the way you used the link element.   All of the changes were good for someone who has not used markdown/github before, as things like adding images were fairly difficult to implement if this was your first time. I also think the way you documented your work was very neat and easy to follow.

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+## Who am I?
```
Comment (manzoniUC) [inline README.md:2]: Good use of varying numbers of "#" symbols to vary the size of the headers.

File: README.md (hunk 1 above)
Comment (manzoniUC) [reply README.md:6]: Great job formatting the alt text in such a way as to not have it read "TEXT HERE" or whatever the automatic thing is. It took me a hot minute to figure out how to make it look like you did. Good job!

File: README.md [hunk 3]
```diff
@@ -1 +1,14 @@
...
+## Who am I? 
+My name is Liam Cohen. I am currently a **rising third year** in the College
+majoring in *Latin American Studies and Political Science*. I highly recommend 
+that you check out the 
+[Center for Latin American Studies](https://clas.uchicago.edu/). 
+There are great professors associated with the Center. 
+
+## academic interests 
+- Democracy 
+- Political (In-)Stability
```
Comment (manzoniUC) [inline README.md:11]: I like the use of a list here!

**Review by manzoniUC — COMMENTED** [review]
Hi Liam, nice to e-meet you. 
I also struggled with learning how to consistently push commits up to github - I eventually figured out (as I assume you did) that I had to tick the README box when hitting commit and push. As for your use of markdown, I think you did a good job incorporating different aspects of the formatting style like headers, text emphasis, lists and the embedding of links. On that last point, I particularly liked that you formatted the alt text in such a way as to make it read seamlessly.
Good job!

File: README.md (hunk 1 above)
Comment (bensoltoff) [inline README.md:6]: If you want all the text in a single paragraph, you don't have to manually split them into separate lines. You can just let the text auto-wrap in the script editor

File: README.md (hunk 3 above)
Comment (bensoltoff) [inline README.md:11]: I recommend leaving a blank line between different Markdown elements, such as between headers and paragraph text. It avoids rendering issues in more complex documents

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |       x    |              |            |
| **Achievement, creativity** |      x     |              |            |
| **Reproducibility**         |      x     |              |            |

**Pull request closed without merging**